gh/workflows: Set LVH image version to 5.10

brb opened the pull request:
The recent change in lvh-action \[1\] allows us to set a custom Kind LVH image version. For now, set it to 5.10. Once others (bpf-next, 4.9, 5.4) are fixed, we can run the DP conformance suite on all of them.

\[1\]: https://github.com/cilium/little-vm-helper-action/commit/9eb481cbe6a32bf74273106b315d43aeaa46a2ee

brb committed:
gh/workflows: Set LVH kind image version to 5.10

The recent change in lvh-action [1] allows us to set a custom Kind LVH
image version. For now, set it to 5.10. Once others (bpf-next, 4.9, 5.4)
are fixed, we can run the DP conformance suite on all of them.

[1]: https://github.com/cilium/little-vm-helper-action/commit/9eb481cbe6a32bf74273106b315d43aeaa46a2ee

Signed-off-by: Martynas Pumputis <m@lambda.lt>

sayboras reviewed (approved):
:shipit: as long as CI is happy :rocket:

brb committed:
gh/workflows: Get status before running DP tests

This might help to explain why the tests are sometimes failing due to:

connectivity test failed: failed to fetch cilium runtime config: unable
to upgrade connection: container not found ("cilium-agent")

Signed-off-by: Martynas Pumputis <m@lambda.lt>
